### DISCUSSION TITLE
Mentor Query

### DISCUSSION THREAD

**Discussion by Aakash1121** [oppia/oppia-android#4152] [Q&A (Other)] [answered]
How can I know, who is my mentor?
Is it mentioned in the assigned issue? or anywhere else.
Also, how can I reach out to my mentor?
Guide me please.

**Comment by MaskedCarrot** [comment, marked as answer]
You have to sign the CLA and fill out the contributor survey form as mentioned [here](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#onboarding-instructions). After that, you will get an email from the mentor assigned to you.

You can reach out to your mentor via email or google chat.
